src/App.jsx

```diff
@@ -14,45 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
+const goodsWithId = getGoodsWithId(goodsFromServer);
+
+function getGoodsWithId(goods) {
+  return goods.map((product, index) => ({ product, id: index + 1 }));
+}
+
+function getSortCallback(sortField) {
+  return (goodA, goodB) => {
+    switch (sortField) {
+      case SORT_FIELD_ALPHABET:
+        return goodA.product.localeCompare(goodB.product);
+
+      case SORT_FIELD_LENGTH:
+        return goodA.product.length - goodB.product.length;
+
+      default:
+        return 0;
```

Comment:
I think there is no need to call the sort method at all if it's the default case